tools/build/__tests__/__snapshots__/compiledImports.test.ts.snap

```diff
@@ -632,63 +632,358 @@ var require_dist = __commonJS({
   }
 });
 
-// node_modules/lodash.camelcase/index.js
-var require_lodash = __commonJS({
-  \\"node_modules/lodash.camelcase/index.js\\"(exports, module) {
-    var INFINITY = 1 / 0;
+// node_modules/lodash/_freeGlobal.js
```

Comment:
thought: looks like the new file is much larger with the new import method. I'm curious what the bundle-size CI check reports.

Reply:
Not too much bigger, as discussed in focus time